src/amo/components/LanguagePicker.scss

```diff
@@ -7,6 +7,8 @@ $selector-padding: 10px;
 }
 
 .LanguagePicker-selector {
+  -moz-appearance: none;
```

Comment:
Actually, you can probably just set `enablePostCssLoader: true` in your local dev config.